workspaces/local-cli/src/shared/local-cli-task-runner.ts

```diff
@@ -73,7 +76,35 @@ export async function LocalTaskSessionWrapper(
   const captureId = uuid.v4();
   const runner = new LocalCliTaskRunner(captureId, paths);
   const session = new CliTaskSession(runner);
-  await session.start(cli, config, taskName);
+
+  const task = config.tasks[taskName];
+
+  if (!task) {
```

Comment:
This is nice, I think you can handle the empty task list case too, in case they haven't specified any yet.